node/tests/reconstruct_test.go

```diff
@@ -96,36 +99,52 @@ func TestFullReconstructFromLights(t *testing.T) {
 		lnodes = 69
 	)
 
-	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
 	t.Cleanup(cancel)
 	sw := swamp.NewSwamp(t, swamp.WithBlockTime(btime))
-	go sw.FillBlocks(ctx, t, bsize, blocks)
+	errCh := make(chan error)
+	go func() {
+		errCh <- sw.FillBlocks(ctx, bsize, blocks)
+	}()
 
-	cfg := node.DefaultConfig(node.Bridge)
-	cfg.P2P.Bootstrapper = true
-	const defaultTimeInterval = time.Second * 10
+	const defaultTimeInterval = time.Second * 5
 	var defaultOptions = []node.Option{
 		node.WithRefreshRoutingTablePeriod(defaultTimeInterval),
 		node.WithDiscoveryInterval(defaultTimeInterval),
 		node.WithAdvertiseInterval(defaultTimeInterval),
 	}
 
-	bridgeConfig := append([]node.Option{node.WithConfig(cfg)}, defaultOptions...)
+	cfg := node.DefaultConfig(node.Full)
 	cfg.P2P.Bootstrapper = true
-	bridge := sw.NewBridgeNode(bridgeConfig...)
+	bridge := sw.NewBridgeNode()
+	addrsBridge, err := peer.AddrInfoToP2pAddrs(host.InfoFromHost(bridge.Host))
+	bootstrapConfig := append([]node.Option{node.WithConfig(cfg)}, defaultOptions...)
+	bootstapFN := sw.NewFullNode(bootstrapConfig...)
+	require.NoError(t, bootstapFN.Start(ctx))
 	require.NoError(t, bridge.Start(ctx))
-	addr := host.InfoFromHost(bridge.Host)
+	addrBootstrapNode := host.InfoFromHost(bootstapFN.Host)
 
-	nodesConfig := append([]node.Option{node.WithBootstrappers([]peer.AddrInfo{*addr})}, defaultOptions...)
+	require.NoError(t, err)
```

Comment:
was this meant to be the errcheck for L120 ?

Reply:
yes